### PR TITLE
feat/HEYUI-80 Accordion component

### PR DESCRIPTION
**Pull request by derek-heycar**
### Description

Describe your changes (or add ticket description) and add relevant screenshots

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change

### How do I test this

- Add steps to test
- in bullet point format
- preferably you can add link to the storybook build in the PR

### Did you remember to take care of the following?

- [ ] `npm i` – for new NPM dependencies.
- [ ] `npm run lint` - to check for linting issues
- [ ] `npm run test` - to run unit tests
- [ ] `npm run test:screenshots` - to run snapshot tests

### New Feature / Bug Fix

- [ ] Run unit tests to ensure all existing tests are still passing.
- [ ] Add new passing unit tests to cover the code introduced by your pr.

Thanks for contributing!

### CODE REVIEW

File: packages/accordion/src/__tests__/Accordion.test.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+      expect(cb).toBeCalled();
+
+      const props = cb.mock.calls[0][0];
+
+      expect(props['data-test-id']).toBe(dataTestId);
+    });
+  });
+
+  describe('Render tests', () => {
+    test('should contain title', () => {
```
Comment (mwagdi) [inline packages/accordion/src/__tests__/Accordion.test.tsx:179]: What's the difference between using `test` and `it` in this case?

File: packages/accordion/src/styles/default.module.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
+@import '../../../themes/src/default.css';
+
+.accordion {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
```
Comment (mwagdi) [inline packages/accordion/src/styles/default.module.css:9]: Not sure if `height: 100%` is needed. This leads to a break when the component is being used in a grid for some reason

File: .storybook/utils/componentsResolver.js [hunk 3]
```diff
@@ -6,23 +6,94 @@ const componentsDir = path.resolve(__dirname, '../../packages');
...
+          descriptionFileData,
+          request: requestPath,
+        } = request;
+
+        if (requestPath && requestPath.match(componentsPrefix)) {
+
+          const componentName = requestPath.replace(`./${componentsPrefix}`, '');
+          const path = `${componentsDir}`;
+
+        console.log('requestPath -->', requestPath);
```
Comment (esraltintas) [inline .storybook/utils/componentsResolver.js:28]: Do we have any reason for these console.logs, if we don't have to, could you clean please?

**Pull request merged**